.github/workflows/main.yml

```diff
@@ -79,6 +80,33 @@ jobs:
           path: ./*
           key: ${{ github.ref }}-${{ github.sha }}-build-release
 
+  build-docs:
+    needs: setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Restore setup
+        uses: actions/cache@v3
+        with:
+          path: ./*
+          key: ${{ github.ref }}-${{ github.sha }}-setup
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+          bundler-cache: true
+
+      - name: Install jekyll
+        run: gem install jekyll
+
+      - name: Run bundle install
+        working-directory: ./docs
+        run: bundle install
+
```

Comment:
Looks like this can be removed as `bundler-cache: true` already runs this command.
```suggestion
```

Reply:
Done 🙏